wordpress-mobile/WordPress-Android
# [Gutenberg] Release v1.0.1

This PR tracks and brings in the v1.0.1release of Gutenberg-mobile

### To test A:

* Open a gutenberg post and verify that the editor opens the post normally

### To test B:

1. With the "Use Block Editor" App Setting turned on, start a new draft
2. Notice the post title not having this glitch: https://github.com/wordpress-mobile/gutenberg-mobile/issues/666

Update release notes:
